test/metabase/pulse/render/body_test.clj

```diff
@@ -7,6 +7,12 @@
             [metabase.pulse.render.test-util :as render.tu]
             [schema.core :as s]))
 
+(use-fixtures :each
```

Comment:
this is weird. Did this not get merged already?

Reply:
oh i see it was added to js-svg-test and copied to body-test here

Reply:
yeah, I ran into it again in this namespace as well